autoload/fzf/vim.vim

```diff
@@ -1362,11 +1362,22 @@ function! s:mark_sink(lines)
   execute 'normal! `'.matchstr(a:lines[1], '\S').'zz'
 endfunction
 
-function! fzf#vim#marks(...)
+function! fzf#vim#marks(initial_marks, ...)
```

Comment:
How about we make the API backward-compatible? That would require keeping the function signature instact and examining what's inside `...`.